<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyScrollWidget</class>
 <widget class="QWidget" name="MyScrollWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>50</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);
border-radius: 5px;
padding: 0px;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="labelAppName">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelAppVer">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelCMTNum">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelLastTime">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btn_data">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(85, 85, 85, 255), stop:1 rgba(120, 120, 120, 255));
border-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
border:2px;
border-radius:5px;
}
QPushButton:hover {
   background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(120, 120, 120, 255), stop:1 rgba(85, 85, 85, 255));
}
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgb(120, 120, 120);
}</string>
     </property>
     <property name="text">
      <string>数据</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_correct">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(85, 85, 85, 255), stop:1 rgba(120, 120, 120, 255));
border-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
border:2px;
border-radius:5px;
}
QPushButton:hover {
   background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(120, 120, 120, 255), stop:1 rgba(85, 85, 85, 255));
}
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgb(120, 120, 120);
}</string>
     </property>
     <property name="text">
      <string>审阅</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_edit">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(85, 85, 85, 255), stop:1 rgba(120, 120, 120, 255));
border-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
border:2px;
border-radius:5px;
}
QPushButton:hover {
   background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(120, 120, 120, 255), stop:1 rgba(85, 85, 85, 255));
}
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgb(120, 120, 120);
}</string>
     </property>
     <property name="text">
      <string>编辑</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_delete">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(85, 85, 85, 255), stop:1 rgba(120, 120, 120, 255));
border-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
border:2px;
border-radius:5px;
}
QPushButton:hover {
   background-color: qlineargradient(spread:repeat, x1:0, y1:1, x2:1, y2:0, stop:0.272727 rgba(120, 120, 120, 255), stop:1 rgba(85, 85, 85, 255));
}
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgb(120, 120, 120);
}</string>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
